setup/questions.sh

```diff
@@ -77,9 +77,16 @@ address, so we're suggesting $DEFAULT_PRIMARY_HOSTNAME.
 		$DEFAULT_PRIMARY_HOSTNAME \
 		PRIMARY_HOSTNAME
 
+	RE='^.+\.localdomain$'
```

Comment:
These three regular expressions could do with better names or comments

Reply:
I will add comments. The regular expression above checks if the hostname is of the form `*.localdomain`. The two below check if it is a valid FQDN.